doc/releasenotes.html

```diff
@@ -59,7 +59,120 @@ <h2>**CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical
+</ul>
+
+<h2>Battery Storage (2022.11.21)</h2>
+<ul>
+<li>Improve reporting of AC and DC in kW and kWh inputs and outputs. (<a href="https://github.com/NREL/SAM/pull/1183">SAM 1183</a>, <a href="https://github.com/NREL/ssc/pull/899">SSC 899</a>, <a href="https://github.com/NREL/ssc/pull/889">SSC 889</a>)</li>
+<li>Improve passing of SAM inputs to REopt. (<a href="https://github.com/NREL/SAM/pull/1165">SAM 1165</a>)</li>
+<li>Improve transformer loss calculations so losses are better accounted for in battery dispatch. (<a href="https://github.com/NREL/ssc/pull/936"></a>)</li>
+<li>For PV smoothing, change battery state of charge (SOC) to end of time step to be consistent with other dispatch options. (<a href="https://github.com/NREL/ssc/pull/904">SSC 904</a>)</li>
+<li>Restore PV priority option for manual dispatch option. (<a href="https://github.com/NREL/ssc/pull/900">SSC 900</a>)</li>
+<li>Various Levelized Cost of Storage (LCOS) fixes. (<a href="https://github.com/NREL/ssc/pull/883">SSC 883</a>, <a href="https://github.com/NREL/ssc/issues/853">SSC 853</a>)</li>
+<li>Fix price signals dispatch option for rates with different periods in a month. (<a href="https://github.com/NREL/ssc/pull/882">SSC 882</a>)</li>
+<li>Make front-of-meter (FOM) automated dispatch aware of retail electricity prices. (<a href="https://github.com/NREL/ssc/issues/824">SSC 824</a>)</li>
+<li>Improvements to front-of-meter (FOM) automated dispatch. (<a href="https://github.com/NREL/ssc/pull/877">SSC 877</a>)</li>
+<li>New 'utilityrateforecast' compute module for Software Development Kit (SDK) and PySAM coders. (<a href="https://github.com/NREL/ssc/pull/868">SSC 868</a>)</li>
+</ul>
+
+<h2>Concentrating Solar Power (2022.11.21)</h2>
+<ul>
+<li>Remove boiler pressure input from Rankine cycle model. (<a href="https://github.com/NREL/SAM/pull/1192">SAM 1192</a>)</li>
+</ul>
+
+<h2>Marine Energy</h2>
+<ul>
+<li>Add battery storage option to Marine Wave model. (<a href="https://github.com/NREL/ssc/pull/890">SSC 890</a>)</li>
+</ul>
+
+<h2>Geothermal Power (2022.11.21)</h2>
+<ul>
+<li>Update to latest version of GETEM. (<a href="https://github.com/NREL/SAM/pull/1174">SAM 1174</a>, <a href="https://github.com/NREL/ssc/pull/886">SSC 886</a>)</li>
+<li>Fix number of wells calculation. (<a href="https://github.com/NREL/ssc/pull/875">SSC 875</a>)</li>
+</ul>
+
+<h2>Wind Power (2022.11.21)</h2>
+<ul>
+<li>Remove BOS model options from Residential and Commercial configurations. (<a href="https://github.com/NREL/SAM/pull/1169">SAM 1169</a>)</li>
+</ul>
+
+<h2>Fuel Cell</h2>
+<ul>
+<li>Fix issue with fuel cell capex table interpolation. (<a href="https://github.com/NREL/SAM/pull/1272">SAM 1272</a>)</li>
+<li>Allow AC-connected battery to charge from fuel cell. (<a href="https://github.com/NREL/ssc/pull/880">SSC 880</a>)</li>
```

Comment:
Should read "Allow AC-connected battery to charge from fuel cell during grid outage"

Apologies for the misleading title

Reply:
&#128077;